newton/_src/usd/utils.py

```
@@ -146,6 +149,32 @@ def has_applied_api_schema(prim: Usd.Prim, schema_name: str) -> bool:
+    Returns:
+        True when the prim has no face-based ``UsdGeom.Subset`` children but does carry more
+        than one ``st``-prefixed primvar (i.e. ``st`` plus at least one ``st_N`` sibling).
+    """
+    if not prim or not prim.IsValid():
+        return False
+    for child in prim.GetChildren():
+        try:
+            if child.IsA(UsdGeom.Subset):
+                return False
+        except Exception:
+            continue
```

Ruff (0.15.14)

[error] `try`-`except`-`continue` detected, consider logging the exception

(S112)